test/truffle/cexts/backtraces/expected.txt

```diff
@@ -0,0 +1,17 @@
+/test/truffle/cexts/backtraces/bin/backtraces:8:in `block in bar': error (RuntimeError)
+	from /src/main/ruby/core/truffle/cext.rb:1140:in `call'
+	from /src/main/ruby/core/truffle/cext.rb:1140:in `__send__'
+	from /src/main/ruby/core/truffle/cext.rb:1140:in `rb_funcall'
+	from backtraces.c:4:in `baz'
+	from /src/main/ruby/core/truffle/cext.rb:204:in `execute'
+	from /src/main/ruby/core/truffle/cext.rb:204:in `execute_with_mutex'
+	from /src/main/ruby/core/truffle/cext.rb:1209:in `block in rb_proc_new'
+	from /test/truffle/cexts/backtraces/bin/backtraces:8:in `call'
+	from /test/truffle/cexts/backtraces/bin/backtraces:8:in `bar'
+	from /src/main/ruby/core/truffle/cext.rb:1140:in `__send__'
+	from /src/main/ruby/core/truffle/cext.rb:1140:in `rb_funcall'
+	from backtraces.c:8:in `foo'
+	from /src/main/ruby/core/truffle/cext.rb:204:in `execute'
+	from /src/main/ruby/core/truffle/cext.rb:204:in `execute_with_mutex'
+	from /src/main/ruby/core/truffle/cext.rb:1385:in `foo'
+	from /test/truffle/cexts/backtraces/bin/backtraces:13:in `<main>'
```

Comment:
Alternatively, maybe all cext.rb frames should be hidden, as they are implementation details?